test/getRelatedEmployees.test.js

```diff
@@ -0,0 +1,31 @@
+
+const { isManager, getRelatedEmployees } = require('../src/getRelatedEmployees')
+
+describe('4 - Crie uma função que faça a gestão dos funcionários de acordo com o respectivo gerente', () => {
```

Comment:
```suggestion
describe('4 - Crie a gestão dos funcionários de acordo com o respectivo gerente', () => {
```
Fiz essa alteração pois na realidade são duas funções, veja o que faz sentido.